scripts/build-index.js

```diff
@@ -25,12 +25,27 @@ function buildPagesIndex(files) {
       if (!index[page].platform.includes(os)) {
         index[page].platform.push(os);
       }
-
       if (!index[page].language.includes(language)) {
```

Comment:
I would really suggest adding an empty line above L28, L31, L33, and L39. Having this all condensed makes it kind of hard to read.